Rubberduck.Parsing/Annotations/FolderAnnotation.cs

```diff
@@ -13,11 +13,7 @@ public sealed class FolderAnnotation : AnnotationBase
             IEnumerable<string> parameters)
             : base(AnnotationType.Folder, qualifiedSelection)
         {
-            if (parameters.Count() != 1)
-            {
-                throw new InvalidAnnotationArgumentException(string.Format("{0} expects exactly one argument, the folder, but none or more than one were passed.", this.GetType().Name));
-            }
-            _folderName = parameters.First();
+            _folderName = parameters.FirstOrDefault() ?? string.Empty;
```

Comment:
I know. But shouldn't `annotation.FolderName.StartsWith("Foo")` throw a `NullReferenceException` when the folder annotation isn't properly parameterized?